doc/neomake.txt

```diff
@@ -332,8 +332,12 @@ or warnings and the empty string otherwise.
 These functions get the counts of errors by error type for the |location-list|
 and the |quickfix| respectively. The return value is something like this: >
     {'E': 2, 'W': 1, 'x': 5}
-<Where 'E', 'W' and 'x' are error types. Empty error types are ignored for
-now.
+<Where 'E', 'W' and 'x' are error types. Empty error types are ignored for now.
+
+By default, *LoclistCounts* returns the counts for the current buffer (i.e.
+|bufnr(%)|). *LoclistCounts* also takes an optional |buf| argument: passing a
```

Comment:
`|bufnr(%)|` is not a valid link, and not valid Vimscript.
Let's use `bufnr('%')` there (in backticks, without the link `|`).  I can fix this when merging.